src/core/CSSLint.js

```diff
@@ -13,7 +13,7 @@ var CSSLint = (function(){
 
     var rules           = [],
         formatters      = [],
-        embeddedRuleset = /\/\*csslint([^\*]*)\*\//,
+        embeddedRuleset = /\/\*\s?csslint([^\*]*)\*\//,
```

Comment:
Why not allow more than one space in the embedded ruleset?  (i.e. use "*" instead of "?")

Also, it would be cool if you could add a test to `tests/core/CSSLint.js` to make sure the behaviour works the way you expect.  :wink:  Something like:

    "Spaces should be allowed in embedded rulesets": function(){
        var result = CSSLint.verify("/* csslint bogus, adjoining-classes:true, box-sizing:false */\n.foo.bar{}", {
          "text-indent": 1,
          "box-sizing": 1
        });

        Assert.areEqual(2, result.ruleset["adjoining-classes"]);
        Assert.areEqual(1, result.ruleset["text-indent"]);
        Assert.areEqual(0, result.ruleset["box-sizing"]);
    }
would work.